Common/src/containers/CTrapezoidalMap.cpp

```diff
@@ -0,0 +1,232 @@
+/*!
+ * \file TrapezoidalMap.cpp
+ * \brief Implementation of the trapezoidal map for tabulation and lookup of fluid properties
+ * \author D. Mayer, T. Economon
+ * \version 7.3.1 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2022, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "../../Common/include/option_structure.hpp"
+#include "../../Common/include/containers/CTrapezoidalMap.hpp"
+
+using namespace std;
+
+/* Trapezoidal map implementation. Reference: 
+ * M. de Berg, O. Cheong M. van Kreveld, M. Overmars, 
+ * Computational Geometry, Algorithms and Applications pp. 121-146 (2008)
+ */
+CTrapezoidalMap::CTrapezoidalMap(const su2double* samples_x, const su2double* samples_y, const unsigned long size,
+                                 vector<vector<unsigned long> > const& edges,
+                                 vector<vector<unsigned long> > const& val_edge_to_triangle) {
+  int rank = SU2_MPI::GetRank();
+  clock_t build_start = clock();
```

Comment:
Use MPI::Wtime(), I had this in a previous comment already.
The best strategy is to go over the PR comments and mark conversations as resolved after applying the fix, and looking for other places that may need the same change.

Reply:
Sorry about that, I thought I pushed this change already.